packages/SwingSetRunner/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "SwingSetRunner",
+  "version": "0.1.0",
```

Comment:
we might add a `private: true` to this, to make it clear that we're not yet ready to think about publish this to NPM

also the `name` will need to be updated to lowercase, probably `swingset-runner`